src/main/kotlin/uk/gov/justice/digital/hmpps/jobsboard/api/jobs/application/ExpressionOfInterestDeleter.kt

```diff
@@ -13,7 +12,7 @@ class ExpressionOfInterestDeleter(
   private val expressionOfInterestRepository: ExpressionOfInterestRepository,
 ) {
 
-  @Transactional
+  // @Transactional
```

Comment:
temporarily comment this out?

Reply:
I left that comment by mistake. I'll remove it.